Description: fix mingw32 linker error
 without this patch, we get a problem cross-building for mingw32, where the linker script is not recognized.
 .
 This is a similar issue to the one the libgpg-error had in
 https://lists.gnupg.org/pipermail/gnupg-devel/2016-February/030798.html
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Last-Update: 2016-02-14


--- a/src/libksba.def
+++ b/src/libksba.def
@@ -1,3 +1,4 @@
+EXPORTS
 ;; libksba.def - Exported symbols for W32
 ;;      Copyright (C) 2007 g10 Code GmbH
 ;;
@@ -28,7 +29,6 @@
 ;; if not, see <http://www.gnu.org/licenses/>.
 
 
-EXPORTS
       ksba_check_version  @1
 
       ksba_set_hash_buffer_function @2
